go.mod

```diff
@@ -152,36 +151,36 @@ require (
+	k8s.io/api => k8s.io/api v0.30.3
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.30.3
+	k8s.io/apimachinery => k8s.io/apimachinery v0.30.3
+	k8s.io/apiserver => k8s.io/apiserver v0.30.3
+	k8s.io/cli-runtime => k8s.io/cli-runtime v0.30.3
+	k8s.io/client-go => k8s.io/client-go v0.30.3
+	k8s.io/cloud-provider => k8s.io/cloud-provider v0.30.3
+	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.30.3
+	k8s.io/code-generator => k8s.io/code-generator v0.30.3
 	k8s.io/component-base => k8s.io/component-base v0.31.0-alpha.2 // HACK: Workaround https://github.com/kubernetes/kubernetes/issues/124687 by using alpha tag
-	k8s.io/component-helpers => k8s.io/component-helpers v0.30.2
-	k8s.io/controller-manager => k8s.io/controller-manager v0.30.2
-	k8s.io/cri-api => k8s.io/cri-api v0.30.2
-	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.30.2
-	k8s.io/dynamic-resource-allocation => k8s.io/dynamic-resource-allocation v0.30.2
-	k8s.io/endpointslice => k8s.io/endpointslice v0.30.2
-	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.30.2
-	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.30.2
-	k8s.io/kube-proxy => k8s.io/kube-proxy v0.30.2
-	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.30.2
-	k8s.io/kubectl => k8s.io/kubectl v0.30.2
-	k8s.io/kubelet => k8s.io/kubelet v0.30.2
-	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.30.2
-	k8s.io/metrics => k8s.io/metrics v0.30.2
-	k8s.io/mount-utils => k8s.io/mount-utils v0.30.2
-	k8s.io/node-api => k8s.io/node-api v0.30.2
-	k8s.io/pod-security-admission => k8s.io/pod-security-admission v0.30.2
-	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.30.2
-	k8s.io/sample-cli-plugin => k8s.io/sample-cli-plugin v0.30.2
-	k8s.io/sample-controller => k8s.io/sample-controller v0.30.2
+	k8s.io/component-helpers => k8s.io/component-helpers v0.30.3
+	k8s.io/controller-manager => k8s.io/controller-manager v0.30.3
+	k8s.io/cri-api => k8s.io/cri-api v0.30.3
+	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.30.3
+	k8s.io/dynamic-resource-allocation => k8s.io/dynamic-resource-allocation v0.30.3
+	k8s.io/endpointslice => k8s.io/endpointslice v0.30.3
+	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.30.3
+	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.30.3
+	k8s.io/kube-proxy => k8s.io/kube-proxy v0.30.3
+	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.30.3
+	k8s.io/kubectl => k8s.io/kubectl v0.30.3
+	k8s.io/kubelet => k8s.io/kubelet v0.30.3
+	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.30.3
+	k8s.io/metrics => k8s.io/metrics v0.30.3
+	k8s.io/mount-utils => k8s.io/mount-utils v0.30.3
+	k8s.io/node-api => k8s.io/node-api v0.30.3
+	k8s.io/pod-security-admission => k8s.io/pod-security-admission v0.30.3
+	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.30.3
+	k8s.io/sample-cli-plugin => k8s.io/sample-cli-plugin v0.30.3
+	k8s.io/sample-controller => k8s.io/sample-controller v0.30.3
```

Comment:
These should all be bumped to `v0.31.0` to match the other k8s dependencies we have.